src/AzureStack/FabricAdmin/Fabric.Admin/Properties/AssemblyInfo.cs

```diff
@@ -25,5 +25,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("0.1.0.0")]
-[assembly: AssemblyFileVersion("0.1.0.0")]
+[assembly: AssemblyVersion("0.1.1.0")]
```

Comment:
Please do not update assembly version when you change AssemblyFileVersion.
Keep it as 0.1.0.0

Reply:
Done.